src/content/docs/mobile-monitoring/new-relic-mobile-flutter/monitor-your-flutter-application.mdx

```diff
@@ -11,7 +11,7 @@ isTutorial: true
 
 import mobileFlutterSummary from 'images/mobile_screenshot-full_flutter-summary.webp'
 
-Monitor your Flutter application with our New Relic Flutter agent. Capture Dart errors, track specific network requests, use distributed tracing to drill down into handled exceptions, and create custom tracking events and metrics to fully capture your end users' experience.
+Monitor your Flutter-based mobile app with our Flutter agent. Capture Dart errors, track specific network requests, use distributed tracing to drill down into handled exceptions, and create custom tracking events and metrics to fully capture your end users' experience.
```

Comment:
Let's parallel all the hybrid doc titles and the first few sentences. We currently have all kinds of variety between them.

A templated version could look something like...

Title: "Monitor your <hybrid-name> mobile app"

Intro:

Monitor your <framework-name> mobile applications with our <hybrid-name> agent. Once you install and configure our agent, you'll get insights into your mobile app performance, what causes errors, and how to fix them." 

Our <hybrid-name> agent is a powerful tool that can help you monitor your <hybrid-name> mobile apps and improve their performance. Once you install and configure the agent, you will get insights into:

* **Performance**: [How well your app is performing](/docs/mobile-monitoring/new-relic-mobile/get-started/introduction-mobile-monitoring/#speed-performance).
* **Crash insights**: [Why your app crashes and how to prevent them](/docs/mobile-monitoring/new-relic-mobile/get-started/introduction-mobile-monitoring/#crash-analysis)
* **Monitor backend issues**: [How backend activity correlates with performance](/docs/mobile-monitoring/mobile-monitoring-ui/network-pages/analyze-network-requests-using-mobilerequest-event-data/)

The <hybrid-name> agent is easy to install and configure. Once it is installed, it will start collecting data about your app's performance and errors. This data is then sent to our cloud-based platform, where you can view it in real time. You can also [create alerts](/docs/mobile-monitoring/new-relic-mobile/get-started/mobile-monitoring-alert-information/) so that you are notified when there are performance or error issues.